output.go

```diff
@@ -0,0 +1,212 @@
+package timescaledb
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/jackc/pgtype"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/sirupsen/logrus"
+
+	"go.k6.io/k6/output"
+	"go.k6.io/k6/stats"
+)
+
+func init() {
+	output.RegisterExtension("timescaledb", newOutput)
+}
+
+func newOutput(params output.Params) (output.Output, error) {
+	config, err := getEnvConfig(params.Environment)
+	if err != nil {
+		return nil, fmt.Errorf("problem parsing config: %w", err)
+	}
+
+	if config.ConcurrentWrites <= 0 {
+		return nil, fmt.Errorf("concurrent writes must be a positive number")
+	}
+
+	pconf, err := pgxpool.ParseConfig(config.URL)
+	if err != nil {
+		return nil, fmt.Errorf("TimescaleDB: Unable to parse config: %w", err)
+	}
+
+	pool, err := pgxpool.ConnectConfig(context.Background(), pconf)
+	if err != nil {
+		return nil, fmt.Errorf("TimescaleDB: Unable to create connection pool: %w", err)
+	}
+
+	thresholds := make(map[string][]*dbThreshold)
+	for name, t := range params.ScriptOptions.Thresholds {
+		for _, threshold := range t.Thresholds {
+			thresholds[name] = append(thresholds[name], &dbThreshold{id: -1, threshold: threshold})
+		}
+	}
```

Comment:
You should just use [WithThresholds](https://github.com/grafana/k6/blob/b705acb12bc478bf6993ddd2941762077c804ef1/output/types.go#L77-L82) as it's done in the [cloud output](https://github.com/grafana/k6/blob/b705acb12bc478bf6993ddd2941762077c804ef1/output/cloud/output.go#L310)

Reply:
Done